README.md

```diff
@@ -1,176 +1,89 @@
 # PyConTW ETL
+
 ![Python CI](https://github.com/pycontw/PyCon-ETL/workflows/Python%20CI/badge.svg)
 ![Docker Image CI](https://github.com/pycontw/PyCon-ETL/workflows/Docker%20Image%20CI/badge.svg)
 
-Using Airflow to implement our ETL pipelines
+Using Airflow to implement our ETL pipelines.
 
+## Table of Contents
 
-[TOC]
+- [Prerequisites](#prerequisites)
+- [Installation](#installation)
+- [Configuration](#configuration)
+- [BigQuery (Optional)](#bigquery-optional)
+- [Running the Project](#running-the-project)
+  - [Local Environment with Docker](#local-environment-with-docker)
+  - [Production](#production)
+- [Contact](#contact)
 
-## Year to Year Jobs
+## Prerequisites
 
-這三個 job 什麼時候打開需要人工確認（麻煩當年的組長大大了），理論上是售票開始前我們要測試一下然後打開
+- [Python 3.8+](https://www.python.org/downloads/release/python-3811/)
+- [Docker](https://docs.docker.com/get-docker/)
+- [Git](https://git-scm.com/book/zh-tw/v2/%E9%96%8B%E5%A7%8B-Git-%E5%AE%89%E8%A3%9D%E6%95%99%E5%AD%B8)
+- [Poetry](https://python-poetry.org/docs/#installation) (Optional, only for creating virtual environments during development)
```

Comment:
Should we add instruction on how to `virtualenv` instead? or use tools like `pip-tools`, `Pipenv`. AFAIK, `poetry` does not use `pip` to resolve dep. I would suggest we either use `poetry` for all env or not use it at all

Reply:
ok, I use venv module instead.